manifest-from-input-json/processJson.py

```diff
@@ -131,14 +138,14 @@ def buildManifest(self):
     def printManifest(self):
         print(json.dumps(self.result_json, indent=2))
 
-    def _write_json_s3(self, key, data):
+    def _write_json_s3(self):
```

Comment:
We need these args. The writeEventData function calls this _write_json_s3 passing different data